jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -317,7 +317,11 @@ properties:
           msa_configuration_file:
             title: MSA configuration file name
             type: string
-            fits_keyword: MSACONFG
+            fits_keyword: MSACONFL
+          msa_metadata_id:
+            title: MSA metadata ID
+            type: number
```

Comment:
Will having the type set to "number" return the meta attribute as a float? (I think it will). If so, we should define it as integer.